build/common.sh

```diff
@@ -97,35 +97,27 @@ kube::build::get_docker_wrapped_binaries() {
           kube-controller-manager,busybox
           kube-scheduler,busybox
           kube-proxy,gcr.io/google_containers/debian-iptables-amd64:v3
-          federation-apiserver,busybox
-          federation-controller-manager,busybox
```

Comment:
Nope. This is one of the reasons why our distribution size is ridiculously large. We ship the binaries and also the corresponding docker wrapped binaries both the in the distribution/release. It's not necessary. As part of our release process we push hyperkube images to gcr.io/google_containers/hyperkube-<ARCH>:<VERSION> and people can pull the docker images from there. And if someone really can't hit this registry endpoint, they can always just do a 

```
FROM busybox
ADD hyperkube /usr/local/bin/hyperkube
```

and build the image.